src/components/PrimaryButton.tsx

```diff
@@ -39,16 +39,17 @@ const link = css`
     }
 `;
 
-export const CallToAction: React.FC<{
+type Props = {
     url: string;
     linkText: string;
-}> = ({ url, linkText }) => {
-    return (
-        <a className={link} href={url} target="_blank">
-            {linkText}
-            <svg xmlns="http://www.w3.org/2000/svg">
-                <path d="M22.8 14.6l-7.6-7.6-.7.7 5.5 6.6h-14v1.5h14l-5.5 6.6.7.7 7.6-7.6v-.9"></path>
-            </svg>
-        </a>
-    );
 };
+
+export const PrimaryButton: React.FC<Props> = ({ url, linkText }: Props) => (
+    // eslint-disable-next-line react/jsx-no-target-blank
+    <a className={linkStyles} href={url} target="_blank">
```

Comment:
I know not added here, but even though we're always opening another Guardian page here, would it be overkill to add `rel="noopener noreferrer"` to future proof ourselves because of [security concerns with `target="blank"`](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)?

Reply:
I was only slightly worried that adding `rel="noopener noreferrer"` could lead to any unexpected behaviour in terms of affiliates/tracking/analytics, but didn't find any evidence to support that. So let's add it and monitor once released.